cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -234,6 +234,7 @@
    */
   public static final class Dataset {
 
+    public static final String CDAP_VERSION = "cdap.version";
```

Comment:
I thought we were not adding (new) non-cdap-site parameter names to this class? Is there another place where this could go?